extensions.json

```
@@ -455,8 +455,8 @@
     {
       "id": "hediet.vscode-drawio",
       "repository": "https://github.com/hediet/vscode-drawio",
-      "version": "0.7.2",
-      "checkout": "v0.7.2",
+      "version": "999.0.0-alpha",
+      "checkout": "master",
```

The latest release tag from that repo is still `v0.7.2`.

However, even under that tag, the `package.json` `version` is indeed `999.0.0-alpha`, which caused the OpenVSX upload to have that version as well: https://open-vsx.org/extension/hediet/vscode-drawio

It seems like we're missing some kind of `"prepublish"` build or packaging or versioning command.

Looking at [the `package.json` "scripts"](https://github.com/ayu-theme/vscode-ayu/blob/48eab0dc3d9ea4bcf209cd355337977ddbd7a5d1/package.json#L70-L75), I think we should run `npm run build` and/or `npm run package` in our `"prepublish"` step.

@spoenemann Do you agree?

---

The package script just runs `vsce package`, which is already done by `ovsx`.

---

Wait sorry, that was the wrong `package.json` file. [Here is the correct one](https://github.com/hediet/vscode-drawio/blob/14209ad462028cbf7013bc8e094bf0fc6e811d30/package.json#L272-L280).

@spoenemann so, do you think we should add anything else to our `"prepublish"` step (which already has `yarn pre-build`)?

Meanwhile, maybe let's undo these changes:

```suggestion
      "version": "0.7.2",
      "checkout": "v0.7.2",
```

---

@jankeromnes Done!